euphoria-testing/src/main/java/cz/seznam/euphoria/testing/AbstractFlowTest.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2016-2017 Seznam.cz, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.seznam.euphoria.testing;
+
+import cz.seznam.euphoria.core.client.dataset.Dataset;
+import cz.seznam.euphoria.core.client.flow.Flow;
+import cz.seznam.euphoria.core.client.io.ListDataSink;
+import cz.seznam.euphoria.core.executor.Executor;
+import cz.seznam.euphoria.executor.local.LocalExecutor;
+
+import java.util.List;
+
+/**
+ * Abstract test class for user's {@link Flow} testing.
+ *
+ * @param <OUT> type of output dataset
+ */
+public abstract class AbstractFlowTest<OUT> {
+
+  /**
+   * This method describes how the final dataset should look like.
+   *
+   * @return expected output
```

Comment:
We should mention here, that the final list is compared without respect to ordering.

Reply:
👍